ScottWu.md

```diff
@@ -9,18 +9,20 @@
 - [Date-time Picker]
 	- [Bug 1283381](https://bugzilla.mozilla.org/show_bug.cgi?id=1283381) - [meta] Implement UI for date time input types
 		- [Bug 1283382](https://bugzilla.mozilla.org/show_bug.cgi?id=1283382) - Implement UI for \<input type="month"\>
-			- Improved the structure of the month picker
-			- Working on putting together the month picker and run it in Firefox
 	- [Tracking] [Bug 1280654](https://bugzilla.mozilla.org/show_bug.cgi?id=1280654) - Expose a generic API for polling data out of CLDR
-		- Continued discussion about the APIs
+	- [Tracking] [Bug 1287677](https://bugzilla.mozilla.org/show_bug.cgi?id=1287677) - Expose an API to provide display names for calendar terms
+		- Gave Zibi feedback on the APIs
```

Comment:
With the latest comment, we need to figure out if there is a chrome-only alternative. Maybe you could comment on the bug to ask if we should. We cannot afford waiting for a public JavaScript API to be discussed and designed.